lib/helperModels.ts

```diff
@@ -1,6 +1,7 @@
 /*
  *  Copyright 2016 Alexander Tsybulsky and other contributors
  *  Copyright 2020 Centrapay and other contributors
+ *  Copyright 2023 Stitch and other contributors
```

Comment:
Not sure who to speak to about copyright on forked packages

Reply:
I mentioned it to Barney in the TL sync. I'll follow through on it